src/gpodder/gtkui/main.py

```diff
@@ -1711,6 +1711,7 @@ def filter_by_activity(activity, tasks):
         elif finished_syncs:
             message = self.format_episode_list([str(task) for task in finished_syncs])
             self.show_message(message, _('Device synchronization finished'))
+            gpodder.user_extensions.on_all_episodes_synced()
```

Comment:
If you look above, this will only get called when all syncs finish, if there's any failure it won't get called (even if some syncs finished, and others failed -- the `if finished_syncs and failed_syncs` above).

Maybe call it unconditionally or something?

Reply:
Makes sense, moved it one conditional above, where it will run in case of both successful and failed sync. Please let me know if anything else would make sense from gPodder's development perspective!